theme/src/js/helpers/api-helpers.js

```diff
@@ -8,26 +8,46 @@ const httpRequest = axios.create(
 	}
 );
 
-export function getArticlesByPerPageCount(id, locale, count) {
-	return httpRequest.get(
-		`help_center/${locale}/sections/${id}/articles.json?per_page=${count}`
-	);
-}
+/**
+ * Returns a promise of Articles under a Section
+ * @param {string} id - The section id
+ * @param {string} locale - The user's current locale
```

Comment:
We can remove the `-` between the parameter name and the description.

Reply:
Sure thing, will make the change in the next pr.